src/univariate/continuous/chisq.jl

```diff
@@ -70,6 +70,13 @@ function entropy(d::Chisq)
     hν + logtwo + loggamma(hν) + (1 - hν) * digamma(hν)
 end
 
+function kldivergence(p::Chisq, q::Chisq)
+    pν = dof(p)
+    qν = dof(q)
+    return kldivergence(Chi(pν), Chi(qν))
```

Comment:
Maybe we could change this line to
```suggestion
    return kldivergence(Chi{typeof(pν)}(pν), Chi{typeof(qν)}(qν))
```
to avoid checks and promotions.